src/StatusQ/Core/StatusIdenticonRingSettings.qml

```diff
@@ -0,0 +1,11 @@
+import QtQuick 2.13
+import StatusQ.Core 0.1
+
+QtObject {
+    id: statusIdenticonRingSettings
+
+    property var ringSpec // Expected ListModel object to iterate through ring segments
```

Comment:
```suggestion
    property var ringSpecModel // [ListElement {colorId: 0; segmentLength: 1}, ...]
```
Wouldn't it be more readable than a plain text comment? Also added `Model` postfix to the name of the variable.

Reply:
I like it! :)